stratum/hal/lib/bcm/bcm_switch.cc

```diff
@@ -401,6 +402,18 @@ ::util::Status BcmSwitch::RetrieveValue(uint64 /*node_id*/,
         resp.mutable_node_packetio_debug_info()->set_debug_string(
             "A (sample) node debug string.");
         break;
+
+      case DataRequest::Request::kOpticalChannelInfo: {
+        ::util::Status status = phal_interface_->GetOpticalTransceiverInfo(
+            req.optical_channel_info().node_id(),
+            req.optical_channel_info().port_id(),
+            resp_val.mutable_optical_channel_info());
+        if (status.ok()) {
+          resp = resp_val;
+        }
+        break;
```

Comment:
It can be `status.update(...)`. `case DataRequest::Request::kPortCounters` is an example (line 333)

Reply:
Changed. Please review.